### PR TITLE
Ovirt vms improve

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

ovirt_vms

##### ANSIBLE VERSION

```
2.3.0
```

##### SUMMARY

 - add serial_policy parameter to vms module
 - add timezone parameter to vms module
 - add comment parameter to vms module
 - add description parameter to vms module
 - use 'and' at begging of next line
 - use param method instead of module.params
 -  add instance type parameter
 - add function to get id by name

### CODE REVIEW

**Comment by machacekondra** [comment]
shipit

**Comment by machacekondra** [comment]
@mwperina @ryansb

**Review by mwperina — APPROVED** [review]
Looks good to me, thanks! 👍

File: lib/ansible/modules/cloud/ovirt/ovirt_vms.py [hunk 1]
```diff
@@ -246,6 +247,35 @@
...
+        version_added: "2.3"
+    serial_policy:
+        description:
+            - "Specify a serial number policy for the Virtual Machine."
+            - "Following options are supported:"
+            - "C(vm) - Sets the Virtual Machine's UUID as its serial number."
+            - "C(host) - Sets the host's UUID as the Virtual Machine's serial number."
+            - "C(custom) - Allows you to specify a custom serial number in C(serial_policy_value)."
+    serial_policy_value:
+        description:
```
Comment (abadger) [inline lib/ansible/modules/cloud/ovirt/ovirt_vms.py]: Do serial_policy and serial_policy_value need version_added 2.3?

File: lib/ansible/modules/cloud/ovirt/ovirt_vms_facts.py [hunk 2]
```diff
@@ -51,6 +51,16 @@
...
+    all_content:
+      description:
+        - "If I(true) all the attributes of the virtual machines should be
+           included in the response."
+    case_sensitive:
+      description:
+        - "If I(true) performed search will take case into account."
+    max:
+      description:
+        - "The maximum number of results to return."
```
Comment (abadger) [inline lib/ansible/modules/cloud/ovirt/ovirt_vms_facts.py:63]: version_added: "2.3" needed for these as well?

Comment (abadger) [reply lib/ansible/modules/cloud/ovirt/ovirt_vms_facts.py:63]: Ah, I see that this module was added in 2.3 so not needed here.

**Comment by abadger** [comment]
Community modules, owner_pr and reviewed by another person in the ovirt community.  This is almost ready to merge.

I noticed some documentation that probably needs changing.  Once that's fixed, we can merge this in.

File: lib/ansible/modules/cloud/ovirt/ovirt_vms.py (hunk 1 above)
Comment (machacekondra) [reply lib/ansible/modules/cloud/ovirt/ovirt_vms.py]: Done.

File: lib/ansible/modules/cloud/ovirt/ovirt_vms_facts.py (hunk 2 above)
Comment (machacekondra) [reply lib/ansible/modules/cloud/ovirt/ovirt_vms_facts.py:63]: Yes, it's there since 2.3 :)

**Comment by machacekondra** [comment]
Thanks for review.

**Review by mwperina — APPROVED** [review]
👍

**Review by ryansb — APPROVED** [review]

**Pull request merged**